lib/faraday/rack_builder.rb

```diff
@@ -73,7 +73,7 @@ def initialize_dup(original)
     def build
       raise_if_locked
       block_given? ? yield(self) : request(:url_encoded)
-      adapter(Faraday.default_adapter) unless @adapter
+      adapter(Faraday.default_adapter, Faraday.default_adapter_options) unless @adapter
```

Comment:
Have you tried the double splat operator?

```suggestion
      adapter(Faraday.default_adapter, **Faraday.default_adapter_options) unless @adapter
```

Reply:
Thank you! I've just fixed it!（ ec72df9 ）